examples/release_highlights/plot_release_highlights_0_23_0.py

```diff
@@ -142,7 +142,8 @@
 plot_partial_dependence(gbdt_cst, X, features=[0],
     line_kw={'linewidth': 4, 'label': 'constrained'}, ax=disp.axes_)
 disp.axes_[0, 0].plot(X[:, 0], y, 'o', alpha=.5, zorder=-1, label='samples')
-disp.axes_[0, 0].set_ylim(-3, 3); disp.axes_[0, 0].set_xlim(-1, 1)
+disp.axes_[0, 0].set_ylim(-3, 3)
+disp.axes_[0, 0].set_xlim(-1, 1)
```

Comment:
So now the only change is this bit, and we tend to not include cosmetic only PRs.

I'm very happy you're finding sklearn an interesting package to contribute to, but I'd suggest starting with a good first documentation issue maybe? I'll close this PR, but very happy to have you around and see you work on other issues :)